packages/components/src/sandbox/index.native.js

```diff
@@ -230,14 +230,14 @@ function Sandbox( {
 					className={ type }
 				>
 					<div dangerouslySetInnerHTML={ { __html: html } } />
-					<script
-						type="text/javascript"
-						dangerouslySetInnerHTML={ {
-							__html:
-								customJS ||
-								`(${ observeAndResizeJS.toString() })();`,
-						} }
-					/>
+					{ customJS && (
```

Comment:
Keeping the ability to inject `customJS` but I haven't verified if an custom script will work on Android. I'm fine with removing this since it's not verified injecting scripts this way will work. 

Instead I can add the `customJS` override in the `injectedJavaScript` prop although I'm not sure why we would want the core resize events to be overridden.

Reply:
Probably we could also inject the `customJS` via the `injectedJavaScript`, instead of adding it to the HTML. Especially if we know that can have issues when running on Android.

> Instead I can add the customJS override in the injectedJavaScript prop although I'm not sure why we would want the core resize events to be overridden.

Actually, there's a case for WordPress embeds:
https://github.com/WordPress/gutenberg/blob/23392e8228c8a19b6e546959514a5b7bfb9e3f01/packages/block-library/src/embed/wp-embed-preview.native.js#L7-L51
https://github.com/WordPress/gutenberg/blob/23392e8228c8a19b6e546959514a5b7bfb9e3f01/packages/block-library/src/embed/wp-embed-preview.native.js#L73

So, I understand that we'll have the same issue on Android for that type of embed.

Reply:
In case you want to test a WP embed, you can use the following URL: `https://wordpress.org/news/2021/07/tatum/`.

Reply:
> Actually, there's a case for WordPress embeds

Oh awesome, thanks for pointing that out.

> So, I understand that we'll have the same issue on Android for that type of embed.

Good news is that custom JS does not call `toScript()`.  It _might_ work in the script tag but since it seems the intent of `customJS` or at least the expectation is to override the built in resize handler script. `Sandbox` could probably use a better prop name like `observeAndResizeScript` but we can save that for another day.